linked_list/LinkedList.cpp

```diff
@@ -0,0 +1,423 @@
+/*
+ * C++ Program to Implement Singly Linked List
+ */
+#include<iostream>
+#include<cstdio>
+#include<cstdlib>
+using namespace std;
+/*
+ * Node Declaration
+ */
+struct node
+{
+    int info;
+    struct node *next;
+}*start;
+
+/*
+ * Class Declaration
+ */
+class single_llist
+{
+    public:
+        node* create_node(int);
+        void insert_begin();
+        void insert_pos();
+        void insert_last();
+        void delete_pos();
+        void sort();
+        void search();
+        void update();
+        void reverse();
+        void display();
+        single_llist()
+        {
+            start = NULL;
+        }
+};
+
+/*
+ * Main :contains menu
+ */
+main()
+{
+    int choice, nodes, element, position, i;
+    single_llist sl;
+    start = NULL;
+    while (1)
+    {
+        cout<<endl<<"---------------------------------"<<endl;
+        cout<<endl<<"Operations on singly linked list"<<endl;
+        cout<<endl<<"---------------------------------"<<endl;
+        cout<<"1.Insert Node at beginning"<<endl;
+        cout<<"2.Insert node at last"<<endl;
+        cout<<"3.Insert node at position"<<endl;
+        cout<<"4.Sort Link List"<<endl;
+        cout<<"5.Delete a Particular Node"<<endl;
+        cout<<"6.Update Node Value"<<endl;
+        cout<<"7.Search Element"<<endl;
+        cout<<"8.Display Linked List"<<endl;
+        cout<<"9.Reverse Linked List "<<endl;
+        cout<<"10.Exit "<<endl;
+        cout<<"Enter your choice : ";
+        cin>>choice;
```

Comment:
Hardcode all the things don't take user input, it is the cause of travis build failure.